### PR TITLE
[Manifest] (RequestedHeight) Correct examples

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Updated ranges and ordering based on info in [mailappversionoverrides/1.1 schema](https://docs.microsoft.com/en-us/openspecs/office_file_formats/ms-owemxml/8e722c85-eb78-438c-94a4-edac7e9c533a).

### CODE REVIEW

**Comment by o365devx** [comment]
Docs Build status updates of commit _[4eb0fbc](https://github.com/OfficeDev/office-js-docs-pr/commits/4eb0fbc448e8210911bd311ca2ecc3e65652f4be)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/contextual-outlook-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/outlook/contextual-outlook-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/extensionpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/extensionpoint.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/requestedheight.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/requestedheight.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/requestedwidth.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/requestedwidth.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004091744391098-1732%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6TrQXMFfT9HR6JOJBw%2FZoxQGU8ogrmLUEQC6dyfl%2FnM%3D&st=2020-04-09T17%3A40%3A50Z&se=2020-05-10T17%3A45%3A50Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/manifest/desktopsettings.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ Specifies source location and control settings that apply when your mail add-in
 <Form xsi:type="ItemRead">
    <!--website.html is a placeholder for your own add-in website.-->
    <DesktopSettings>
-      <SourceLocation DefaultValue="https://website.html" />
-      <!--RequestedHeight must be between 240px to 800px, inclusive.-->
+      <!--If you opt to include RequestedHeight, it must be between 32px to 450px, inclusive.-->
       <RequestedHeight>360</RequestedHeight>
+      <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/desktopsettings.md:25]: I don't think this is a valid URL.
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

File: docs/reference/manifest/desktopsettings.md [hunk 2]
```diff
@@ -20,14 +20,14 @@ Specifies source location and control settings that apply when your mail add-in
...
    </DesktopSettings>
    <TabletSettings>
-      <SourceLocation DefaultValue="https://website.html" />
-      <!--RequestedHeight must be between 240px to 800px, inclusive.-->
+      <!--If you opt to include RequestedHeight, it must be between 32px to 450px, inclusive.-->
       <RequestedHeight>360</RequestedHeight>
+      <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/desktopsettings.md:30]:
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

File: docs/reference/manifest/desktopsettings.md [hunk 3]
```diff
@@ -20,14 +20,14 @@ Specifies source location and control settings that apply when your mail add-in
...
    </TabletSettings>
    <PhoneSettings>
       <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/desktopsettings.md:33]:
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

File: docs/reference/manifest/tabletsettings.md [hunk 4]
```diff
@@ -20,14 +20,14 @@ Specifies control settings that apply when your mail add-in is used on a tablet.
 <Form xsi:type="ItemRead">
    <!--website.html is a placeholder for your own add-in website.-->
    <DesktopSettings>
-      <SourceLocation DefaultValue="https://website.html" />
-      <!--RequestedHeight must be between 240px to 800px, inclusive.-->
+      <!--If you opt to include RequestedHeight, it must be between 32px to 450px, inclusive.-->
       <RequestedHeight>360</RequestedHeight>
+      <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/tabletsettings.md:25]:
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

File: docs/reference/manifest/tabletsettings.md [hunk 5]
```diff
@@ -20,14 +20,14 @@ Specifies control settings that apply when your mail add-in is used on a tablet.
...
    </DesktopSettings>
    <TabletSettings>
-      <SourceLocation DefaultValue="https://website.html" />
-      <!--RequestedHeight must be between 240px to 800px, inclusive.-->
+      <!--If you opt to include RequestedHeight, it must be between 32px to 450px, inclusive.-->
       <RequestedHeight>360</RequestedHeight>
+      <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/tabletsettings.md:30]:
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

**Comment by ElizabethSamuel-MSFT** [comment]
Related to issue https://github.com/OfficeDev/office-js/issues/1074

File: docs/reference/manifest/tabletsettings.md [hunk 6]
```diff
@@ -20,14 +20,14 @@ Specifies control settings that apply when your mail add-in is used on a tablet.
...
    </TabletSettings>
    <PhoneSettings>
       <SourceLocation DefaultValue="https://website.html" />
```
Comment (Rick-Kirkham) [inline docs/reference/manifest/tabletsettings.md:33]:
```suggestion
      <SourceLocation DefaultValue="https://MyDomain.com/website.html" />
```

**Comment by o365devx** [comment]
Docs Build status updates of commit _[f9035c3](https://github.com/OfficeDev/office-js-docs-pr/commits/f9035c34bc20d532f5ebf984f7ae58067e3784ab)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/outlook/contextual-outlook-add-ins.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/outlook/contextual-outlook-add-ins.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/desktopsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/desktopsettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/extensionpoint.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/extensionpoint.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/form.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/form.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/phonesettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/phonesettings.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/requestedheight.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/requestedheight.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/requestedwidth.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/requestedwidth.md) | :white_check_mark:Succeeded | |
[docs/reference/manifest/tabletsettings.md](https://github.com/OfficeDev/office-js-docs-pr/blob/elizs-requested/docs/reference/manifest/tabletsettings.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C4%5C9%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C202004091759394251-1732%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=pYEvS9V8bJBnISRf7e63pHVezVrbCA20luGcAF71pXA%3D&st=2020-04-09T17%3A55%3A51Z&se=2020-05-10T18%3A00%3A51Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by ElizabethSamuel-MSFT** [comment]
Also received offline signoff from Outlook engineer.

**Pull request merged**